src/clean.js

```diff
@@ -1,6 +1,6 @@
 // eslint-disable-next-line no-unused-vars
 const clean = (ast, newObj, parent) => {
-  ['code', 'codeStart', 'loc', 'range'].forEach(name => {
+  ['code', 'codeStart', 'loc', 'operations', 'range'].forEach(name => {
```

Comment:
this is in order to bypass the problem w/ `AST_COMPARE=true`

Reply:
This is no longer necessary. Merging master into this branch fixes the issue.